api/model.go

```diff
@@ -176,10 +176,11 @@ type Event struct {
 
 type SupportBundle struct {
 	client.Resource
-	NodeID       string                     `json:"nodeID"`
-	State        manager.BundleState        `json:"state"`
-	Name         string                     `json:"name"`
-	ErrorMessage manager.BundleErrorMessage `json:"errorMessage"`
+	NodeID                     string                     `json:"nodeID"`
+	State                      manager.BundleState        `json:"state"`
+	Name                       string                     `json:"name"`
+	ErrorMessage               manager.BundleErrorMessage `json:"errorMessage"`
+	DownloadProgressPercentage int                        `json:"progress"`
```

Comment:
Just call it `Progress` or `ProgressPercentage`. Keep the JSON variable name the same (except the first capitalized character).

And it's not `Download`.